backend/app/src/main/resources/db/migration/V2__seed_data_in_tables.sql

```diff
@@ -377,63 +331,99 @@ insert into TABLE_COLUMN_INFO(TABLE_NAME, COLUMN_NAME, HEADER_NAME, VARIABLE_NAM
 );
 
 insert into TABLE_COLUMN_INFO(TABLE_NAME, COLUMN_NAME, HEADER_NAME, VARIABLE_NAME) VALUES (
-'PROCESS_DATA', 'HDS_HOURS_OF_OPERATION', 'HDS_HOURS_OF_OPERATION', 'hdsHoursOfOperation'
+'PROCESS_DATA', 'HDS_STATUS', 'HDS_STATUS', 'hdsStatus'
+);
+insert into TABLE_COLUMN_INFO(TABLE_NAME, COLUMN_NAME, HEADER_NAME, VARIABLE_NAME) VALUES (
+'PROCESS_DATA', 'SOURCE_STATUS', 'SOURCE_STATUS', 'sourceStatus'
 );
 
 insert into TABLE_COLUMN_INFO(TABLE_NAME, COLUMN_NAME, HEADER_NAME, VARIABLE_NAME) VALUES (
-'PROCESS_DATA', 'HDS_CONTACT_NAME', 'HDS_CONTACT_NAME', 'hdsContactName'
+'PROCESS_DATA', 'PCN_CLINIC_STATUS', 'PCN_CLINIC_STATUS', 'pcnClinicStatus'
 );
 
 insert into TABLE_COLUMN_INFO(TABLE_NAME, COLUMN_NAME, HEADER_NAME, VARIABLE_NAME) VALUES (
-'PROCESS_DATA', 'HDS_IS_FOR_PROFIT_FLAG', 'HDS_IS_FOR_PROFIT_FLAG', 'hdsIsForProfitFlag'
+'PROCESS_DATA', 'HDS_EFFECTIVE_START_DATE', 'HDS_EFFECTIVE_START_DATE', 'hdsEffectiveStartDate'
 );
 
 insert into TABLE_COLUMN_INFO(TABLE_NAME, COLUMN_NAME, HEADER_NAME, VARIABLE_NAME) VALUES (
-'PROCESS_DATA', 'SOURCE_STATUS', 'SOURCE_STATUS', 'sourceStatus'
+'PROCESS_DATA', 'PLR_FACILITY_ID', 'PLR_FACILITY_ID', 'plrFacilityId'
+);
+
+
+insert into TABLE_COLUMN_INFO(TABLE_NAME, COLUMN_NAME, HEADER_NAME, VARIABLE_NAME) VALUES (
+'PROCESS_DATA', 'FAC_RELN_TYPE', 'FAC_RELN_TYPE', 'facRelnType'
```

Comment:
can we use full names ?

Reply:
Default values are set as per business requirements. These default values will be sent to PLR.

Reply:
The column names are provided and specified as per business requirement by Linda and Lindsay.